src/openswathalgo/CMakeLists.txt

```diff
@@ -48,17 +48,20 @@ endif()
 # get information regarding the openswath files
 include(source/OPENSWATHALGO/OpenSwathAlgoFiles.cmake)
 
+#------------------------------------------------------------------------------
+# get information regarding the MIToolbox files
+set(MITOOLBOX_INCLUDE_DIRECTORY ../openms/thirdparty/MIToolbox/include)
```

Comment:
These two variables are also set by the ``CMakeLists.txt`` file of ``openms/thirdparty``, however ``OpenSwathAlgo`` is built before ``OpenMS``. Is this the only way or is there a cleaner solution?

Reply:
Hmm yes. I think it would be cleaner the way it is done in eol_bspline. It basically creates a new target for that library and exports its include directories. Then I think both OpenSwathAlgo and OpenMS could refer to it.

Reply:
I also liked the eol_bspline approach much better and tried to implement this. But then I ran into a circular issue, because the OpenMS target needs to be configured first to build MIToolbox, but also depends on OpenSwathAlgo, which depends on MIToolbox of OpenMS.

Is there a way to include the MIToolbox target in OpenSwathAlgo prior to configuring the OpenMS target?

Reply:
Hmm I see. Why does it need OpenMS to be configured? Because it is in the OpenMS subfolder? In that case I think it would be fine to just make an openswathalgo/thirdparty folder.

Reply:
Exactly, at least I couldn't figure out how to overcome the issue. I agree that putting it to `openswathalgo/thirdparty` might be a good option.